scripts/utils/gdx_rename.sh

```diff
@@ -71,7 +71,7 @@ if (length(args) == 1) {
     c_iterative_target_adj = 'cm_iterative_target_adj', 
     c_co2_tax_2020         = 'cm_co2_tax_2020', 
     c_macscen              = 'c_macscen', 
-    c_bioenergy_tax        = 'cm_bioenergy_tax', 
+    c_bioenergy_tax        = 'cm_bioenergy_SustTax',
```

Comment:
Shouldn't we keep this line and add a new one with:
`cm_bioenergy_tax = 'cm_bioenergy_SustTax',`

Reply:
I changed it.